### DISCUSSION TITLE
Discord community

### DISCUSSION THREAD

**Discussion by richwomanbtc** [nkaz001/hftbacktest#91] [General]
How about creating a server in discord? It would make it easier for users to make suggestions and ask simple questions.

**Comment by nkaz001** [comment]
so far, i haven't created a discord server because i don't think i can respond to messages in a timely manner. maybe it's time to set one up, but i still might not be very responsive.
https://discord.gg/K9pDsWrPEe

**Reply by richwomanbtc** [reply]
Thank you for your quick response.
I believe this is a good framework and I hope we can discuss it, including contributions.
